internal/cli/atlas/kubernetes/operator/install.go

```diff
@@ -148,15 +148,24 @@ func InstallBuilder() *cobra.Command {
 		Args:    require.NoArgs,
 		Aliases: cli.GenerateAliases(use),
 		Short:   "Install Atlas Kubernetes Operator to a cluster.",
-		Long:    `This command installs one of the supported versions of Atlas Kubernetes Operator to an existing cluster, as well as automatically import Atlas resources to be managed by the operator.`,
+		Long: `This command installs a supported versions of Atlas Kubernetes Operator to an existing cluster, and optionally imports Atlas resources to be managed by the operator.
```

Comment:
singular, plural conflict:
```suggestion
		Long: `This command installs a supported version of Atlas Kubernetes Operator to an existing cluster, and optionally imports Atlas resources that are managed by the operator.
```